client/src/components/AuthForm/SignUpForm/SignUpForm.js

```diff
@@ -92,32 +122,29 @@ const SignUpForm = ({
             content={buttonText}
             withButton
             icon={<CheckIcon />}
+            classInput={
+              signUpForm.password !== ''
+                ? styles.activeInput
+                : styles.defaultInput
+            }
+            classSpan={
+              signUpForm.password !== ''
+                ? styles.activeInputSpan
+                : styles.defaultInputSpan
+            }
           />
           <Button
             text="Sign Up"
             disabled={checkDisabled()}
+            isLoading={!loading ? false : isLoading}
             btnRole="primary btnBlock"
           />
-          {alert.length > 0 ? <Alert alerts={alert[0].message} /> : null}
+          {alert.message === '' ? null : <Alert alerts={alert.message} />}
```

Comment:
here too

Reply:
actually here it can be a refactored a little differently

`          {alert.message && <Alert alerts={alert.message} />}
`